deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/ibm-block-csi-operator.v1.6.0.clusterserviceversion.yaml

```diff
@@ -122,11 +122,170 @@ spec:
   displayName: "Operator for IBM block storage CSI driver"
   description: |
     The Container Storage Interface (CSI) Driver for IBM block storage systems enables container orchestrators such as Kubernetes to manage the life cycle of persistent storage.
-
     This is the official operator to deploy and manage IBM block storage CSI driver.
+    ## Supported orchestration platforms
+    The following table details orchestration platforms suitable for deployment of the IBM® block storage CSI driver.
+    |Orchestration platform|Version|Architecture|
+    |----------------------|-------|------------|
+    |Kubernetes|1.20|x86|
+    |Kubernetes|1.21|x86|
+    |Red Hat® OpenShift®|4.7|x86, IBM Z®, IBM Power Systems™|
+    |Red Hat OpenShift|4.8|x86, IBM Z, IBM Power Systems|
+
+     1 IBM Power Systems architecture is only supported on Spectrum Virtualize Family storage systems.
+
+    **Note:** As of this document's publication date, IBM Cloud® Satellite only supports RHEL 7 on x86 architecture for Red Hat OpenShift. For the latest support information, see [cloud.ibm.com/docs/satellite](https://cloud.ibm.com/docs/satellite).
+    ## Supported storage systems
+    IBM® block storage CSI driver 1.6.0 supports different IBM storage systems as listed in the following table.
+    |Storage system|Microcode version|
+    |--------------|-----------------|
+    |IBM FlashSystem™ A9000|12.x|
+    |IBM FlashSystem A9000R|12.x|
+    |IBM Spectrum Virtualize™ Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)|7.8 and above, 8.x|
+    |IBM Spectrum Virtualize as software only|7.8 and above, 8.x|
+    |IBM DS8000® Family|8.x and higher with same API interface|
+
+     **Note:**
+    -   Newer microcode versions may also be compatible. When a newer microcode version becomes available, contact IBM Support to inquire whether the new microcode version is compatible with the current version of the CSI driver.
+    -   The IBM Spectrum Virtualize Family and IBM SAN Volume Controller storage systems run the IBM Spectrum Virtualize software. In addition, IBM Spectrum Virtualize package is available as a deployable solution that can be run on any compatible hardware.
+    ## Supported operating systems
+    The following table lists operating systems required for deployment of the IBM® block storage CSI driver.
+    |Operating system|Architecture|
+    |----------------|------------|
+    |Red Hat® Enterprise Linux® (RHEL) 7.x|x86, IBM Z®|
+    |Red Hat Enterprise Linux CoreOS (RHCOS)|x86, IBM Z®, IBM Power Systems™|
 
-    For compatibility, prerequisites, release notes, and other user information, see the [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver).
+     1 IBM Power Systems architecture is only supported on Spectrum Virtualize Family storage systems.
```

Comment:
the superscript doesn't seem to work
![image](https://user-images.githubusercontent.com/57443811/122252193-d6142d80-ced3-11eb-972e-e30db8902f6a.png)
I think we might have to sync it as we did before, with little to no copy-paste :/